### PR TITLE
docs: explain how JSON Schema files are served

### PR DESCRIPTION
**Pull request by smoya**
**Description**

This PR adds a `README.md` file under `/schemas` directory so we can explain what's the preferable way for downloading/using AsyncAPI JSON Schema documents.

**Related issue(s)**
https://github.com/asyncapi/website/pull/680#issuecomment-1153700681

### CODE REVIEW

File: schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# AsyncAPI JSON Schema definitions
+
+All the JSON Schema documents for validating AsyncAPI documents are located here.
```
Comment (derberg) [inline schemas/README.md:3]: `here` - did you want to add a link or specify here as `here in this repository`?

File: schemas/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# AsyncAPI JSON Schema definitions
+
+All the JSON Schema documents for validating AsyncAPI documents are located here. 
+
+## Download and usage
+Files are served via `https://asyncapi.com/definitions/<asyncapi-version>.json`, for example, https://asyncapi.com/definitions/2.4.0.json. 
+
+In that way, documents are served with the proper `Content-Type` response header. It also allows us to measure AsyncAPI user adoption of those files ([More info in this Github Issue](https://github.com/asyncapi/website/issues/780)).
+We encourage you to **only** download the files from our website instead of from Github.
```
Comment (derberg) [inline schemas/README.md:9]:
```suggestion
We encourage you to **only** download the files from our website instead of from GitHub.
```

**Review by derberg — COMMENTED** [review]
2 tiny comments

@dalelane @fmvilas do you need something more?

@jonaslagoni you probably like it is almost there 😄

File: schemas/README.md (hunk 1 above)
Comment (smoya) [reply schemas/README.md:3]: The second; `here in this repository`. I'm going to clarify it then 👍

**Review by jonaslagoni — APPROVED** [review]
LGTM 👍

**Review by derberg — DISMISSED** [review]

**Review by fmvilas — APPROVED** [review]
LGTM 👍

**Pull request merged**